lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -277,6 +277,9 @@ export abstract class PerformanceClient implements IPerformanceClient {
                         if (current.idTokenSize) {
                             previous.idTokenSize = current.idTokenSize;
                         }
+                        if (current.hasOwnProperty("refreshTokenSize")) {
```

Comment:
Why `hasOwnProperty` here, versus just:

```suggestion
                        if (currentrefreshTokenSize) {
```

Reply:
Would like to be consistent here.

Reply:
Would this attach the size where it is undefined as well? @jasonnutter

Reply:
Since we don't need to attach the metric when it's undefined, I would revert this. Thanks for the suggestion!

Reply:
It would not attach it if `refreshTokenSize` is any falsey value (0, undefined, null, nan, etc)

Reply:
updated!